pkgs/by-name/tl/tlsrouter/package.nix

```diff
@@ -0,0 +1,30 @@
+{
+  buildGo124Module,
+  fetchFromGitHub,
+  lib,
+}:
+
+buildGo124Module rec {
+  pname = "tlsrouter";
+  rev = "c4b9df066048ad2ab5c32235362fa94444a24ebe";
+  version = "${builtins.substring 0 6 rev}";
```

Comment:
```suggestion
  rev = "c4b9df066048ad2ab5c32235362fa94444a24ebe";
  version = "${builtins.substring 0 6 rev}";
```
don't do this. Use version = "version-unstable-date"

Reply:
thx! done it now